docs/release-notes/august-2023.md

```diff
@@ -0,0 +1,157 @@
+---
+slug: august-2023
+append_help_link: true
+title: August 2023
+hide_title: true
+description: >-
+  Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+---
+
+# August 2023 release notes
+
+## Semgrep OSS Engine
+
+:::caution
+Semgrep version 1.38.0 introduces several removals that may break your Semgrep workflows. See [Semgrep OSS > Removed](#removed) for more information.
+:::
+This section of release notes includes upgrades of Semgrep OSS Engine for versions between **1.35.0** and **1.38.3**.
+
+### Added
+
+- The CLI now returns the commit timestamp when running `semgrep ci`.
+- Added optional `min-version` and `max-version` fields for a Semgrep rule, specifying a range of compatible Semgrep versions.
+    - If a rule is incompatible with the version of Semgrep being used, it is reported in the JSON output at the newly-added `info` level, which doesn't cause an exit failure. ([#8496](https://github.com/returntocorp/semgrep/pull/8496/))
+- The `semgrep scan` command is now more resilient to failures when fetching a configuration file (config) from Semgrep servers.
+    - If it can't fetch a config from Semgrep servers it will use backup infrastructure to fetch the most recent successful config for that customers environment. ([#8459](https://github.com/returntocorp/semgrep/pull/8459/))
+- `metavariable-comparison`: You can now use `in` and `not in` for strings in the same sense as in Python, for substring checking. ([#2979](https://github.com/returntocorp/semgrep/pull/8406))
+- **Julia:** Added the deep expression operator. Now you can write patterns such as `foo(<... 42 ...>)` to find instances of calls to `foo` that contain `42` somewhere inside of it. ([#8540](https://github.com/returntocorp/semgrep/pull/8540))
+- Added support for languages with case insensitive identifiers and generalized PHP to use these case insensitive identifiers.
+  - For example, in PHP the pattern `MyClass()` now matches calls with different capitalizations such as `myclass()` and `Myclass()`. ([#8356](https://github.com/returntocorp/semgrep/pull/8356))
+
+### Fixed
+
+- Fixed `--text` and `--output` flags which broke in 1.38.0. If you are using version 1.38.0, update Semgrep to receive these fixes.
+- Version 1.38.0 introduced some regressions with the Semgrep docker image. This is fixed in version 1.38.3.
```

Comment:
This phrasing seems odd to me, because technically there were never regressions with the version tagged 1.38.0 - the problem was with the `latest` tag behavior. But I'm having a hard time coming up with anything better. `Docker` does need to be capitalized though I think.

Reply:
I'll remove this line - the message to UPDATE NAO is also served by the breakage in the output and text flags.